src/Components/HireMeForm/HireMeForm.js

```diff
@@ -113,8 +113,8 @@ export default function ContactUs({
                 required
               >
                 {citiesArray.map((city, index) => (
-                  <Option key={index} value={city.name} name="city">
-                    {city.name}
+                  <Option key={index} value={city.city} name="city">
```

Comment:
this shouldn't be city.city, maybe change the prop name to something else

Reply:
It was city.name but they changed the cities data so I had to change it here in order to work. Should I change the data again?

Reply:
ok then change the prop name to something else so it would be only city
```suggestion
                  <Option key={index} value={city} name="city">
```